apps/web/pages/event-types/[type]/index.tsx

```diff
@@ -254,6 +261,7 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
             periodStartDate: periodDates.startDate,
             periodEndDate: periodDates.endDate,
             periodCountCalendarDays: periodCountCalendarDays === "1",
+            minimumBookingNotice: minimumBookingNoticeInMinutes,
```

Comment:
@om-ray  Locally I still see "0" being saved when I save the minimum booking notice. Could you test this and see if it also occurs on your side?

Reply:
@JeroenReumkens I'll check right now!